tests/providers/common/sql/operators/test_sql.py

```diff
@@ -226,7 +226,7 @@ def test_get_hook(self, mock_get_conn, database):
 
     def test_not_allowed_conn_type(self, mock_get_conn):
         mock_get_conn.return_value = Connection(conn_id='sql_default', conn_type='s3')
-        with pytest.raises(AirflowException, match=r"You are trying to use `common-sql`"):
+        with pytest.raises(AirflowException, match=r'Unknown hook type "s3"'):
```

Comment:
I thought better replace conn_type from `s3`, which not exists anymore to existed conn_type like `aws`, `google_cloud_platform`, `http` so we won't change actual test - provide use unsupported Hook

Reply:
ohhhh... goooood point lemme see